src/index.html

```diff
@@ -1,13 +1,345 @@
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#compare">
+                  Compare Bikes
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#details">
+                  Details
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#contact">
+                  Contacts
+                </a>
+              </li>
+
+            </ul>
+          </nav>
+
+          <a href="tel:+1 234 5555-55-55" class="menu__phone-number">
+          +1 234 5555-55-55
+          </a>
+
+          <a href="tel:+1 234 5555-55-55" class="menu__book">Book a test ride</a>
+        </div>
+      </div>
+
+
+
+    </aside>
+
+
+    <main class="main">
+      <div class="container">
+        <div class="main__content">
+
+          <section class="about_us" id="about_us">
+            <h2 class="
+              about_us__title
+              section_title
+              section_title--first-block
+            ">
+              Move Free
+            </h2>
+            <p class="about_us__description">
+              Shift your ride, not gears. Find the fastest way to move in the city as the bike adapts intuitively to power the speed you need.
+            </p>
+          </section>
+
+          <section class="compare" id="compare">
+            <h2 class="section_title">Compare Bikes</h2>
+            <div class="compare__bikes">
+              <article class="bike compare__product">
+                <a href="#">
+                  <img
+                    src="images/bike_3.png"
+                    alt="bike_3"
+                    class="bike__photo"
+                  >
+                </a>
+                <h3 class="bike__title">
+                  Sporty 4
+                </h3>
+                <p class="bike__description">
+                  The iconic frame brought to a new performance height as a sporty, active ride.
+                </p>
+                <p class="bike__price">$ 2 590</p>
+              </article>
+              <article class="bike compare__product">
+                <a href="#">
+                  <img
+                    src="images/bike_2.png"
+                    alt="bike_2"
+                    class="bike__photo"
+                  >
+                </a>
+                <h3 class="bike__title">
+                  Ride in town ST
+                </h3>
+                <p class="bike__description">
+                  An open frame for an upright riding position as the most comfortable ride in town.
+                </p>
+                <p class="bike__price">$ 2 590</p>
+              </article>
+              <article class="bike compare__product">
+                <a href="#">
+                  <img
+                    src="images/bike_1.png"
+                    alt="bike_1"
```

Comment:
This `alt` is not perfect

Make sure to put something as descriptive as possible - for instance the name of the bike (Bike Agile Ride 3)

Re-check other `alt`s too